robot/EDA/tests/browser/health_check/account_model_check.robot

```diff
@@ -0,0 +1,31 @@
+*** Settings ***
+Documentation   Validates account model health check card
+Resource        robot/EDA/resources/EDA.robot
+Library         cumulusci.robotframework.PageObjects
+...             robot/EDA/resources/SettingsHealthCheckPageObject.py
+
+Suite Setup     Open Test Browser
+Suite Teardown  Capture screenshot and delete records and close browser
+
+*** Test Cases ***
+Verify account model health check settings checks pass
+    [Documentation]         Validates the account model health card is displayed with settings for
+    ...                     Administrative, Household, Default Account Model and Unique account
+    ...                     record type checks.Also verifies the status of each setting.
+    [tags]                  unstable        rbt:high        W-8880346
+    Go to settings health check
+    Current page should be      Home        Settings Health Check
+    Run account model health check settings
+
+*** Keywords ***
+Run account model health check settings
```

Comment:
this keyword is doing the same for an account, affiliation, course connection. Create a keyword with name and dictionary for validation as variables and you can reuse the same keyword in all tests

Reply:
Thanks for the suggestion. I made the change and customized to reuse it for all scenarios.